docs/k0s-single-node.md

```diff
@@ -28,26 +15,18 @@ mkdir -p ${HOME}/.k0s
 
 #### 2. Generate a default cluster configuration
 ```sh
-k0s default-config | tee ${HOME}/.k0s/k0s.yaml
+k0s default-config > k0s.yaml
```

Comment:
If we're creating `${HOME}/.k0s` then we should pipe k0s.yaml to this directory, or at least add the `cd` command to change into this directory.
Alternatively, we can drop this step altogether.

Reply:
I think the config directory part can be skipped, just work in the current dir. It just adds complexity and an extra step.

Also, the cluster config will mostly live in `/etc/k0s/k0s.yaml` or something like that, I don't see why you would put it into `~/.k0s/k0s.yaml` first.